ui/src/components/EntryListItem/EntryListItem.tsx

```diff
@@ -214,7 +214,7 @@ export const EntryItem: React.FC<EntryProps> = ({entry, focusedEntryId, setFocus
                         {entry.src.ip}
                     </span>
                 </Queryable>
-                <span className={`${styles.tcpInfo}`} style={{marginTop: "18px"}}>:</span>
+				<span className={`${styles.tcpInfo}`} style={{marginTop: "18px"}}>{entry.src.port ? ":" : ""}</span>
```

Comment:
Why adding a ternary operator here? Is it possible to `entry.src.port` to be a empty string or `null`? As far as I know it's not possible.

Reply:
When `X-Forwarded-For` is not empty, we change src IP to the IP in `X-Forwarded-For`, and the src port to `""` (`X-Forwarded-For` does not contain any info regarding the port).
https://github.com/up9inc/mizu/pull/606/files#diff-b90c839c6bda9ab1004f6293c5c5f37858c1597e9f6bfcbf2c2c34e4296e51f6R45

This way the IP is propagated to `resolveIP` and to UI without any more code changes.
https://github.com/up9inc/mizu/blob/e4aa94858cd2547fab6e28a9461ecfda6aa6d614/agent/pkg/api/main.go#L120

Reply:
@nimrod-up9 oh I understand now. I think we should do none of those changes to IP and port. Because that means destroying the data. The forwarded request is a different TCP stream. Why altering the reality? To confuse the person who inspects the traffic?

Reply:
I agree ineprincipal, but I had to come up with a quick fix that doen't require a lot of refactoring.

I would love to improve this solution (in a follow up PR). Do you have an idea how we could append additional info in the DB entry? I guess that would require expanding the data model.

Reply:
@nimrod-up9 I'm not sure about the intention of this change. But if the purpose is doing analysis on the traffic (e.g. yet to be introduced features like https://github.com/up9inc/mizu/pull/623 or https://github.com/up9inc/mizu/pull/578) then this `src.ip` and `src.port` replacements shouldn't be done in the dissection layer. Instead it should be done in an upper layer where the TCP stream is displayed as is but the feature treats the TCP stream differently since it's forwarded.

For now, I guess it's OK to do it like this if it's urgent.